pkg/sca/apptypecheck_test.go

```diff
@@ -0,0 +1,152 @@
+package sca_test
+
+import (
+	"testing"
+
+	"github.com/salsadigitalauorg/shipshape/pkg/sca"
+	"github.com/salsadigitalauorg/shipshape/pkg/shipshape"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsDrupalCheck(t *testing.T) {
+	assert := assert.New(t)
+	c := sca.AppTypeCheck{
+		Disallowed:   []string{"drupal"},
+		Threshold:    1,
+		Markers:      make(map[string][]string),
+		Dirs:         make(map[string][]string),
+		Dependencies: make(map[string][]string),
+		Paths:        []string{"./fixtures/drupal"},
+	}
+
+	c.Markers["drupal"] = []string{"use Drupal\\Core\\DrupalKernel;"}
+	c.Dirs["drupal"] = []string{"web", "docroot"}
+	c.Dependencies["drupal"] = []string{"drupal/core-recommended"}
+
+	c.RunCheck()
+	assert.Equal(shipshape.Fail, c.Result.Status)
+	assert.EqualValues(
+		[]string{"drupal detected at ./fixtures/drupal"},
```

Comment:
any reason why the fixtures are not under `testdata`? the latter is a special directory that is ignored by go tooling.

Reply:
No reason — just wasn't aware of that I'll move it around.